GeneratorInterface/Pythia8Interface/plugins/TopRecoilHook.h

```diff
@@ -0,0 +1,138 @@
+// TopRecoilHook.h is a part of the PYTHIA event generator.
```

Comment:
Is this code modified with respect to the one in PYTHIA?

Reply:
It is not officially in Pythia yet, hopefully there will be a simple switch in the future.
But this is what the authors distributed to ATLAS as well.